gitrise.sh

```diff
@@ -61,18 +62,21 @@ while [ $# -gt 0 ]; do
     -t|--test)
         TESTING_ENABLED="true"
         shift
+    ;;
+     -d|--debug)
+        DEBUG="true"
+        shift
     ;;
     *) 
         echo "Invalid option '$1'"
         usage
-        POSITIONAL+=("$1")
         exit 1
     ;;
     esac
 done
 
-# restore positional parameters
-set -- "${POSITIONAL[@]}"
+# Create temp directory if debugging mode enabled
+[ "$DEBUG" == "true" ] && mkdir -p gitrise_temp
```

Comment:
If a gitrise_temp directory exist should we delete it every time the trigger runs.

Reply:
that's a good point. Changed the if statement to check that first.